sound/soc/sof/sof-audio.c

```diff
@@ -881,20 +881,6 @@ struct snd_sof_pcm *snd_sof_find_spcm_comp(struct snd_soc_component *scomp,
 	return NULL;
 }
 
-struct snd_sof_pcm *snd_sof_find_spcm_pcm_id(struct snd_soc_component *scomp,
-					     unsigned int pcm_id)
```

Comment:
remove it from `sof-audio.h` as well?

Reply:
yes, it's not static as I mistakenly thought. fixed now.